src/main/java/seedu/algobase/model/problem/Author.java

```diff
@@ -12,7 +12,13 @@
 
     public static final String MESSAGE_CONSTRAINTS =
             "Author numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+    public static final Author DEFAULT_AUTHOR = new Author();
```

Comment:
You can do something like

`public static final Author DEFAULT_AUTHOR = new Author("");`

instead of creating a new constructor method for Author

Reply:
My solution to creating a default object is: create a private constructor with no arguments, pre-fill the property with values that will be rejected by regex checks in normal constructors. The key is the value can never appear in a non-default instance of this class. `new Author("")` might not work because the constructor might check if the string is empty or not.  Can you take a look at how I implement `DEFAULT_DIFFICULTY` for the Difficulty class? I'm not sure if it's acceptable or not.

Reply:
The validation is done during parsing, so `new Author("")` should not be a problem right?

Reply:
I think `checkArgument(isValidAuthor(author), MESSAGE_CONSTRAINTS);` is also called in the constructor.